lib/pages/profile_page/community_list.dart

```diff
@@ -8,11 +8,25 @@ class CommunityListWidget extends StatelessWidget {
       {super.key,
       required this.user,
       required this.communityLength,
-      required this.token});
+      required this.token,
+      required this.currentEpoch});
```

Comment:
Does `currentEpoch` need to be passed in as a parameter?

Reply:
Yes since the epoch values will be different if they're generated on diff components. 
![image](https://user-images.githubusercontent.com/91294146/200700884-f0f9ddd3-ca6c-40f4-8ff2-9caecf95cbe6.png)